crates/sui-config/src/node.rs

```diff
@@ -182,6 +182,9 @@ pub struct NodeConfig {
 
     #[serde(default)]
     pub execution_cache: ExecutionCacheConfig,
+
+    #[serde(default)]
+    pub state_accumulator_v2: bool,
```

Comment:
shouldn't this be on by default?

Reply:
I was preferring we do a safe rollout, first enabling on mysten validators and seeing in production that everything works/we don't fork. Especially since this PR isn't absolutely critical